plugins/poll/assets/stylesheets/common/poll.scss

```diff
@@ -1,354 +1,438 @@
-div.poll {
-  margin: 1em 0;
-  border: 1px solid var(--primary-low);
-  display: grid;
-  grid-template-areas: "poll" "info" "buttons";
-  @include breakpoint("mobile-extra-large", min-width) {
-    grid-template-columns: 1fr 10em;
-    grid-template-areas: "poll info" "buttons buttons";
-  }
-
-  ul,
-  ol {
-    margin: 0;
-    padding: 0;
-    list-style: none;
-    display: inline-block;
-    width: 100%;
-  }
-
-  li[data-poll-option-id] {
-    color: var(--primary);
-    padding: 0.5em 0;
-    word-break: break-word;
+.topic-body {
+  .cooked {
+    overflow: visible;
```

Comment:
@jjaffeux I want to be very explicit and up-front about this change.

Here is the problem:

* The normal value in core (`hidden`)  prevents the drop-down menu from rendering its contents
* The current widget based dropdown goes to enormous lengths to over come this by using javascript calculated translations.  Now that widgets are legacy it might make sense to review this requirement?
* This gets even more annoying when we introduce IRV as that has dropdown lists in the basic voting UI and those need to be able to be rendered past the borders of the cooked area.

Can we change this overflow value in core to `visible`?   What is it protecting against?

Reply:
I think I'm close to answering my own question here.  I've found this:

[createPopper](https://github.com/discourse/discourse/blob/5aefda1deebb19f9e31dd347c847989997ea68a8/plugins/chat/assets/javascripts/discourse/components/chat-channel-message-emoji-picker.gjs#L4)

So technique is applicable to `.gjs` elements.  I'll have a go!

Reply:
@jjaffeux could you provide some suggestions here so I can expedite?

what is the best way to provide a toggleable menu (like the poll admin menu) that will float above the layout so we can keep the overflow hidden on cooked?  Currently this is being manually built and will not float above.  I've noticed you have `DMenu`  - is that appropriate?

additionally, I'm using `DropdownSelectBox` - we also need to make sure the items float above

Reply:
Yes DMenu is appropriate

Reply:
In progress

Reply:
this is now committed, thanks very much to very significant assistance from @jjaffeux 🙏